os/pm/pm_procfs.c

```diff
@@ -447,43 +460,113 @@ static size_t power_devices_read(FAR struct file *filep, FAR char *buffer, size_
 }
 
 /****************************************************************************
- * Name: power_lock_write
+* 	Name: power_lock_write
```

Comment:
procfs is only for provide information, not for control system.
If you want control something through file system, you should register it's own driver.